fs/bcachefs/opts.c

```diff
@@ -363,15 +373,19 @@ int bch2_parse_mount_opts(struct bch_fs *c, struct bch_opts *opts,
 		bch2_opt_set_by_id(opts, id, v);
 	}
 
+	kfree(copied_opts);
```

Comment:
Instead of duplicating the kfree(), add an out label before this line, and then change the other returns to goto out